src/_layouts/generic.njk

```diff
@@ -57,22 +57,25 @@
   {%- endif %}
 
   <div class="app-page-wrapper">
-    <nav class="app-chapter-navigation" aria-label="Pages in this section">
+    <nav class="app-chapter-navigation">
```

Comment:
The nav still needs an `aria-label`. If you don't want to repeat the words, you could also use an `aria-labelledby` and let it refer to an ID on the heading.